_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: default
+title: Mutate event
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Mutate event processors
+
+Mutate event processors allow you to modify events in Data Prepper. You can use these processors to add entries to an event, delete entries from an event, rename keys in an event, copy values within an event, and convert value types in an event.  The following processors allow you to mutate an event:
+
+* [AddEntries](#addentries)
+* [CopyValues](#copyvalues)
+* [DeleteEntry](#deleteentry)
+* [RenameKey](#renamekey)
+* [ConvertEntry](#convertentry)
+
+## AddEntries
+
+The `AddEntries` processor adds entries to an event.
+
+### Configuration
+
+You can configure the `AddEntries` processor with the following options.
+
+| Option | Required | Description |
+| :--- | :--- | :--- |
+| `entries` | Yes | A list of entries to add to an event. |
+| `key` | Yes | The key of the new entry to be added. |
+| `value` | Yes | The value of the new entry to be added. You can use the following data types: strings, booleans, numbers, null, nested objects, and arrays. |
+| `overwrite_if_key_exists` | No | When set to `true`, if `key` already exists in the event, the existing value is overwritten. The default value is `false`. |
+
+### Usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - add_entries:
+        entries:
+        - key: "newMessage"
+          value: 3
+          overwrite_if_key_exists: true
+  sink:
+    - stdout:
+```
+{% include copy.html %}
+
+
+Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with this filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
+
+For example, before you run the `AddEntries` processor, if the `logs_json.log` file contains the following event record:
+
+```json
+{"message": "hello"}
+```
+
+Then, when you run the `AddEntries` processor using the previous configuration, it parses the message into the following output:
```

Comment:
@asifsmohammed @dlvenable Is "parsing" the correct word here? This applies to several sections through this doc.

Reply:
Changed text to remove parsing.